testing/btest/language/at-if-lambda.zeek

```diff
@@ -9,11 +9,7 @@ event zeek_init()
 	{
 	local make_epoch_result = function(pass_name: string): function(ts: time): time
 		{
-@if ( Version::at_least("4.1") )
```

Comment:
Hmm, this is the main piece of the test case and reproducer from the referenced #2075 :-)

So either we remove the two test parts that use `Version::at_least()`, or place a `TEST-REQUIRES` to skip the test in CPP mode. Slight preference for the latter. Or split it in two and just skip one, but not sure it's worth it.

Reply:
Ah, I missed that the whole point of this test is to have an `@if` inside a lambda! :-P  In that case, yes, a `TEST-REQUIRES` to skip it makes sense.  I thought it was just a now-vestigial test.

Do you want to make the tweak or should I go ahead and do so?

Reply:
I'll pick it up (and if I get it wrong you'll see next time around).